packages/vkui/src/components/Touch/Touch.test.tsx

```diff
@@ -382,5 +347,41 @@ describe('Touch', () => {
       userEvent.click(screen.getByTestId('touch'));
       expect(cb).toBeCalled();
     });
+    it('does not prevent click of a button after slide of a parent on mobile', () => {
```

Comment:
Правильней было бы назвать
```suggestion
    it('should not prevent click to link after scroll with touch', () => {
```

Но вообще тест на текущий момент не покрывает кейс, он работает что до, что и после.

Надо переписать тест так, чтобы он падал при старой реализации `Touch` (до изменений).

Reply:
Тест исправил